### DISCUSSION TITLE
How to include molar volume into Nasa database species

### DISCUSSION THREAD

**Discussion by Noxical** [reaktoro/reaktoro#311] [Q&A]
### Description

Hello,

I am back into Reakoro after long holiday ;)

I have question regarding adding molar volume to nasa database. Currently when using the nasa database, the molar volume is not included (for liquid and solids). 
However, I want to add molar volumes to each species by adding constant Vm value/ or better interpolation valules.
Is it possible to add this to the nasa database via python?
I know I can add the molar volume by creating a new species using StandardThermoModel with interpolation. However in that case I cannot use the Nasa database. 

Regards,
Ruud

### Example

I tried something like:'
```
param = rkt.StandardThermoModelParamsConstant
param.V0 = 7.1e-6
model = rkt.StandardThermoModelConstant(param)
```
 but don't really know how to add to Nasa db species

### What is your operating system?

None

**Comment by allanleal** [comment]
Hi @Noxical, welcome back! Apologies for not replying earlier. For this to work, we would need to add a `V0` member in [StandardThermoModelParamsNasa](https://github.com/allanleal/reaktoro-private/blob/f98cb4a2a7ecf76c67ad9236a49ce315f834f54a/Reaktoro/Models/StandardThermoModels/StandardThermoModelNasa.hpp#L27) and the use this value when computing the standard molar volume of the species at given temperature and pressure. 

The NASA-CEA database does not contain this information, and this motivated this design. But it would be better to have this value in `StandardThermoModelParamsNasa`. Let me know what you did in the meantime to overcome this.

**Reply by Noxical** [reply]
He Allen, thanks. Makes sense. For now I actually assume that solid and liquid volumes are constant and basically do not include volume/pressure variations for solids/liquid. Which actuall for most of the problems within metallurgy is acceptable. But if it will I will post something in this thread to followup ;)

**Reply by allanleal** [reply]
Hi Noxical, thanks for letting me know. I created Issue #317 based on this discussion.
